src/workspace-data/useDataTableProvider.ts

```diff
@@ -33,98 +37,122 @@ export const useDataTableProvider = (
   const [wdsApp, setWdsApp] = useState<LoadedState<ListAppItem, string>>({ status: 'None' });
   const [wdsTypes, setWdsTypes] = useState<LoadedState<RecordTypeSchema[], string>>({ status: 'None' });
   const [wdsCapabilities, setWdsCapabilities] = useState<LoadedState<Capabilities, string>>({ status: 'None' });
+  const [useCwds, setUseCwds] = useState<LoadedState<boolean>>({ status: 'None' });
+  const [wdsUrl, setWdsUrl] = useState<LoadedState<string>>({ status: 'None' });
+  const cwdsURL = getConfig().cwdsUrlRoot;
```

Comment:
how about some code comments for what each of these states represent? Just reading through the code, it gets pretty confusing between `wdsApp`, `wdsUrl`, `useCwds`, etc.